#Exception Handling
#My Practice
print("Enter your first number")
num1=input()
print("Enter your first number")
num2=input()
try:
    print("The sum of two numbers is",int(num1)+int(num2))
except Exception as r:
    print(r)
    print("Aastey")










